Kajabi/sage-lib
# Empty State - Style Update and Prop Deprecation

## Description

Renames the `scope` prop to `size` and updates styles to match [Figma](https://www.figma.com/design/CC1YmaGKHnsvB28yLY9mEH/Mercury-%E2%86%92-Sage-components?node-id=33742-6953&t=mnFVrLyz83wmYLrF-1).

This PR does not include a gradient.

## Screenshots

|  Before  |  After  |
|--------|--------|
<img width="926" alt="Screenshot 2024-07-30 at 4 35 29 PM" src="https://github.com/user-attachments/assets/c0035b88-89a5-4845-9560-607d6637a897">|<img width="1073" alt="Screenshot 2024-08-02 at 2 03 21 PM" src="https://github.com/user-attachments/assets/040f05ae-aa93-4b65-84f9-c743736c3a7e">


## Testing in `sage-lib`

- Navigate to [Rails](http://localhost:4000/pages/component/empty_state?tab=preview) and [React](http://localhost:4100/?path=/docs/sage-emptystate--default)
- Check that component matches Figma specs.

## Testing in `kajabi-products`

1. (**BREAKING**) - Renames the `scope` prop to `size` and updates styles to match Figma


## Related

[DSS-737](https://kajabi.atlassian.net/browse/DSS-737)

[DSS-737]: https://kajabi.atlassian.net/browse/DSS-737?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
